test/stub-test.js

```diff
@@ -2171,6 +2230,16 @@ describe("stub", function () {
             refute.defined(instance.stub());
         });
 
+        it("cleans 'resolvesThis' behavior", function () {
```

Comment:
It is generally better to be very specific when writing expectations, especially in codebases with distributed contributors. Generic expectations often lead to readers trying to derive the intent from the implementation of the test. If there's a mistake in the implementation, then the reader is likely to end up with an incorrect understanding of then intended behaviour.

I would suggest rewriting the expectation to something like this:

```js
it("cleans 'resolvesThis' behavior using stub.resetBehavior", function () {
it("calls stub's resetBehavior method to clean behavior", function () {
```